File: JavalinTest.java
import io.javalin.Javalin;

public class JavalinTest {
    public static void main(String[] args) {
        Javalin app = Javalin.create(config -> {
            // Basic configuration test
        }).get("/", ctx -> ctx.result("Hello Javalin with Kotlin 2.0.21!"))
          .get("/health", ctx -> ctx.json(java.util.Map.of("status", "ok", "kotlin", "2.0.21")))
          .start(7070);
        
        System.out.println("✅ Javalin started successfully with Kotlin 2.0.21!");
        System.out.println("Test server at: http://localhost:7070/");
        System.out.println("Health endpoint: http://localhost:7070/health");
        
        app.stop();
        System.out.println("✅ Javalin stopped successfully!");
    }
}

File: javalin-testtools/src/main/java/io/javalin/testtools/JavalinTest.kt
package io.javalin.testtools

import io.javalin.Javalin

object JavalinTest {

    private val testConfig = TestConfig()
    private val testTool = TestTool(testConfig)

    @JvmStatic
    @JvmOverloads
    fun test(app: Javalin = Javalin.create(), config: TestConfig = this.testConfig, testCase: TestCase) =
        testTool.test(app, config, testCase)

    @JvmStatic
    fun captureStdOut(runnable: Runnable) = testTool.captureStdOut(runnable)

}
